packages/block-library/src/navigation/edit/unsaved-inner-blocks.js

```diff
@@ -118,19 +171,23 @@ export default function UnsavedInnerBlocks( {
 		blocks,
 	] );
 
-	return (
-		<div className="wp-block-navigation__unsaved-changes">
-			<Disabled
+	return createElement(
+		isSaving ? Disabled : 'div', // when saving make component disabled.
+		{
+			className: 'wp-block-navigation__unsaved-changes',
+		},
```

Comment:
Would be good to avoid mixing `createElement` and JSX. I think something like this works:
```jsx
const Wrapper = isSaving ? Disabled : 'div';

return (
    <Wrapper className="wp-block-navigation__unsaved-changes">
        // ...
    </Wrapper>
);

Reply:
I did try that but I was getting an error about `div`. Let me double check I wasn't making a typo late yesterday...